README.md

```diff
@@ -234,72 +247,75 @@ $ deno test
 
 ## Deploying Your App
 
-Once you're done with development, you can deploy the production version of your
-app to Slack hosting using `slack deploy`:
+Once development is complete, deploy the app to Slack infrastructure using
+`slack deploy`:
 
 ```zsh
 $ slack deploy
 ```
 
-After deploying, [create a new link trigger](#create-a-link-trigger) for the
-production version of your app (not appended with `(dev)`). Once the trigger is
-invoked, the workflow should run just as it did in when developing locally.
+When deploying for the first time, you'll be prompted to
+[create a new link trigger](#creating-triggers) for the deployed version of your
+app. When that trigger is invoked, the workflow should run just as it did when
+developing locally (but without requiring your server to be running).
 
-### Viewing Activity Logs
+## Viewing Activity Logs
 
-Activity logs for the production instance of your application can be viewed with
-the `slack activity` command:
+Activity logs of your application can be viewed live and as they occur with the
+following command:
 
 ```zsh
-$ slack activity
+$ slack activity --tail
 ```
 
 ## Project Structure
 
-### `manifest.ts`
+### `.slack/`
 
-The [app manifest](https://api.slack.com/future/manifest) contains the app's
-configuration. This file defines attributes like app name and description.
+Contains `apps.dev.json` and `apps.json`, which include installation details for
+development and deployed apps.
 
-### `slack.json`
+### `datastores/`
 
-Used by the CLI to interact with the project's SDK dependencies. It contains
-script hooks that are executed by the CLI and implemented by the SDK.
+[Datastores](https://api.slack.com/automation/datastores) securely store data
+for your application on Slack infrastructure. Required scopes to use datastores
+include `datastore:write` and `datastore:read`.
 
-### `/external_auth`
+### `functions/`
 
-[External authentication](https://api.slack.com/future/external-auth) connects
-your app to external services using OAuth2. Once connected, you can perform
-actions as the authorized user on these services from a custom function.
```

Comment:
Just checking, but was this section meant to be removed?

Reply:
Absolutely not! Good catch. I entirely forgot this was added. 🤦🏼‍♀️ Are there any other projects that also have this section?

Reply:
No worries! And sorry for a lagging response here! I believe it should also be included on [`deno-github-functions`](https://github.com/slack-samples/deno-github-functions) and [`deno-simple-survey`](https://github.com/slack-samples/deno-simple-survey), even though it's not included on the GitHub functions sample at the moment 😰